PluralKit.Bot/Commands/Message.cs

```diff
@@ -51,15 +51,27 @@ public async Task EditMessage(Context ctx)
         if (!ctx.HasNext())
             throw new PKSyntaxError("You need to include the message to edit in.");
 
+        var originalMsg = await _rest.GetMessageOrNull(msg.Message.Channel, msg.Message.Mid);
+        if (originalMsg == null)
+            throw new PKError("Could not edit message.");
+
+        // Check if we should append or prepend
+        var append = ctx.MatchFlag("append");
+        var prepend = ctx.MatchFlag("prepend");
+        
+        // Grab the original message content and new message content
+        var originalContent = originalMsg.Content;
         var newContent = ctx.RemainderOrNull().NormalizeLineEndSpacing();
+ 
+        // Append or prepend the new content to the original message content if needed.
+        // If no flag is supplied, the new contents will completly overwrite the old contents
+        // If both flags are specified. the message will be prepended AND appended
+        newContent = append ? $"{originalContent} {newContent}" : $"{newContent}";
```

Comment:
this should really not be an inline conditional

Reply:
I changed it from an inline conditional to an if-then statement. Also, the documentation is updated now